src/templates/team-page.js

```diff
@@ -15,20 +15,27 @@ export const TeamPageTemplate = ({
   advisors_section: advisorsSection = {},
 }) => {
   return (
-    <div className="main-content">
+    <div className="team-content">
       <PageSection id={sections.team.HOME}>
         <Row>
           <Col xs={12}>
             <h1 dangerouslySetInnerHTML={{ __html: title }} />
+            <img src={require("../../static/images/svg/green-rect.svg")} />
```

Comment:
@maymax777, You can make the title's underline(green-rect) as following:
```
<h1 className="h1-underline">{hero.title}</h1>
```
So, we don't need `green-rect.svg` file actually.